resources/lang/en.json

```
@@ -102,7 +102,7 @@
 	"filter": "Filter",
 	"filter-closed": "Display only closed resources",
 	"flags": "Flags",
-	"footer-company": "WikiTide Foundation. A registered 501(c)(3) nonprofit company no. 5312868 in the United States",
+	"footer-company": "WikiTide Foundation. A registered 501(c)(3) nonprofit company in the United States. EIN 93-2356406",
```

```suggestion
	"footer-company": "WikiTide Foundation, Inc. - a 501(c)(3) public charity in the United States. EIN: 93-2356406",
```

Minor suggestion to bring at least a little more consistent with the landing page